jobs/mozaggregator_runner.py

```diff
@@ -0,0 +1,10 @@
+# This runner has been auto-generated from mozilla/python_mozaggregator/bin/dataproc.sh.
+# Any changes made to the runner file will be over-written on subsequent runs.
+from mozaggregator import cli
+
+try:
+    cli.entry_point(auto_envvar_prefix="MOZETL")
+except SystemExit:
+    # avoid calling sys.exit() in databricks
```

Comment:
Nit: this probably doesn't hurt, but I think we no longer need this since this is targeting Dataproc.